ios-target/Networking/Services/TargetAPI.swift

```diff
@@ -21,17 +22,24 @@ class TargetAPI {
     }, failure: failure)
   }
   
-  // TODO:
-//  class func createTarget(_ success: @escaping ([Target]) -> Void, failure: @escaping (_ error: Error) -> Void) {
-//    APIClient.request(.get, url: targetsUrl, success: { response, _ in
-//      do {
-//        guard let targets = response["targets"] as? [Any], !targets.isEmpty else {
-//          failure(App.error(domain: .parsing, localizedDescription: "Could not parse valid targets".localized))
-//          return
-//        }
-//        success(Target.parse(targets: targets))
-//      }
-//    }, failure: failure)
-//  }
-  
+  class func createTarget(_ targetTitle: String, targetArea: Double, targetTopic: Int, targetLocation: CLLocationCoordinate2D, success: @escaping (_ response: Target) -> Void, failure: @escaping (_ error: Error) -> Void) {
+    let parameters = [
```

Comment:
you can define a helper in Target to build params, in case you need to do this again later (target edition)

Reply:
cool! didn't think of this!